subsys/bluetooth/host/gatt.c

```diff
@@ -6277,3 +6281,21 @@ void bt_gatt_disconnected(struct bt_conn *conn)
 	remove_cf_cfg(conn);
 #endif
 }
+
+void bt_gatt_req_set_mtu(struct bt_att_req *req, uint16_t mtu)
+{
+	IF_ENABLED(CONFIG_BT_GATT_CLIENT, ({
+		if (req->func == gatt_read_rsp) {
+			struct bt_gatt_read_params *params = req->user_data;
+
+			params->_att_mtu = mtu;
```

Comment:
do we need an `__ASSERT_NO_MSG(params)` here?

Reply:
Added.